test/extensions/filters/network/dubbo_proxy/dubbo_protocol_impl_test.cc

```diff
@@ -151,6 +151,20 @@ TEST(DubboProtocolImplTest, encode) {
   EXPECT_TRUE(dubbo_protocol.decodeData(buffer, context, output_metadata));
 }
 
+TEST(DubboProtocolImplTest, HeartBeatResponseTest) {
+  MessageMetadata metadata;
+  metadata.setMessageType(MessageType::HeartbeatResponse);
+  metadata.setResponseStatus(ResponseStatus::Ok);
+  metadata.setSerializationType(SerializationType::Hessian2);
+  metadata.setRequestId(100);
+
+  Buffer::OwnedImpl buffer;
+  DubboProtocolImpl dubbo_protocol;
+  EXPECT_TRUE(dubbo_protocol.encode(buffer, metadata, ""));
```

Comment:
Looks like an rpc response type is missing in `dubbo_protocol.encode()`.

Reply:
Thanks.